src/cpp/r/R/Api.R

```diff
@@ -641,26 +643,53 @@
 
 .rs.addApiFunction("documentNew", function(type,
                                            code,
-                                           row = 0,
-                                           column = 0,
+                                           row = -1L,
+                                           column = -1L,
```

Comment:
Do we need to worry about existing API usage that calls documentNew w/o specifying row and column? (Will it still do the same thing?)